app/router.js

```diff
@@ -23,6 +26,7 @@ module.exports = app => {
     })
   })
   app.use(offenderRoot, getOffenderDetails)
-  app.get(offenderRoot, (req, res) => res.render('app/index/index'))
+  app.get(offenderRoot, (req, res) => res.redirect(`/individual-id/${req.params.id}/plans`))
```

Comment:
Do we need to redirect? could we not just add the route in below?
app.get([offenderRoot , `${offenderRoot}/plans`], sentencePlanSummary)

Reply:
agreed